docs/installation.md

```diff
@@ -277,38 +277,46 @@ sudo dnf copr enable rhcontainerbot/podman-next -y
 sudo dnf install podman
 ```
 
-## Building from scratch
+## Building from Source
 
 ### Build and Run Dependencies
 
 **Required**
 
-Fedora, CentOS, RHEL, and related distributions you should try to run
-`make package-install` which will install dependencies, build the source,
-produce rpms for the current platform and install them in the end.
+On Fedora:
 
 ```bash
-sudo yum install -y \
-  btrfs-progs-devel \
+# Install build dependencies
+sudo dnf -y builddep rpm/podman.spec
+
+# Install runtime dependencies
+sudo dnf -y install catatonit conmon containers-common-extra
+```
+
+On CentOS Stream 8, enable the PowerTools repository.
+
+On RHEL 8, enable the Appstream and CodeReady Builder (CRB) repositories.
+
+On CentOS Stream 9 and RHEL 9, enable the CodeReady Builder (CRB) repository.
```

Comment:
It might be nice to have examples on how to ready those repos

Reply:
ack, doing it now.